app/javascript/components/SelectCountry/SelectCountry.test.js

```diff
@@ -59,6 +59,6 @@ describe('localisation', () => {
       'SweetSelect'
     );
     const countries: Country[] = select.props().options;
-    expect(countries[2].label).toEqual('Algerien');
+    expect(countries[2].label).toEqual('Åland');
```

Comment:
Interesting... did using Intl.Collator re-sort things the way they were? Or does this mean it didn't work?